archive/100-theorems-list/37_solution_of_cubic.lean

```diff
@@ -0,0 +1,199 @@
+/-
+Copyright (c) 2021 Jeoff Lee. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jeoff Lee
+-/
+import tactic.basic
+import data.complex.basic
+import algebra.group.defs
+
+/-!
+# The Solution of Cubic
+
+This file proves Theorem 37 from the [100 Theorems List](https://www.cs.ru.nl/~freek/100/).
+
+The theorem `cubic_eq_zero_iff` gives solution to the cubic equation over ℂ,
+`cubic_eq_zero_iff_of_discrim_eq_zero`
+based on the [Cardano's Formula](https://en.wikipedia.org/wiki/Cubic_equation#Cardano's_formula).
```

Comment:
I'm not clear what this is saying about the relation between `cubic_eq_zero_iff` and `cubic_eq_zero_iff_of_discrim_eq_zero`
```suggestion
The theorem `cubic_eq_zero_iff` gives the solution to the cubic equation over ℂ,
`cubic_eq_zero_iff_of_discrim_eq_zero`
based on the [Cardano's Formula](https://en.wikipedia.org/wiki/Cubic_equation#Cardano's_formula).
```